.github/workflows/create_change_set_workflow.yml

```diff
@@ -0,0 +1,19 @@
+name: Create Change Set
+
+on:
+  pull_request:
+    types: [synchronize, opened, edited]
```

Comment:
I think we don't need the `synchronize` option here. Just the options `opened` ns `edited`. The `synchronize` option triggers the workflow when new commits are pushed to the PR, but we don't require that. Fragments should only be created/updated when the metadata of the PR (i.e title or description) is created/updated.

@ashwin-pc, @BSFishy, @joshuarrrr, @ananzh  what do you think?

Reply:
Yep lets remove that then :)

Reply:
Code updated! Thanks Ashwin!